lms/djangoapps/commerce/utils.py

```diff
@@ -32,3 +39,29 @@ def audit_log(name, **kwargs):
     message = u'{name}: {payload}'.format(name=name, payload=payload)
 
     log.info(message)
+
+
+class EcommerceService(object):
+    """ Helper class for ecommerce service integration. """
+    def __init__(self):
+        self.config = CommerceConfiguration.current()
+
+    def is_enabled(self):
+        """ Check if the service is enabled and that it's not a microsite. """
+        return self.config.checkout_on_ecommerce_service and not helpers.is_request_in_themed_site()
+
+    def payment_page_url(self):
+        """ Return the URL for the checkout page. Example:
+            http://localhost:8002/basket/single_item/
+        """
+        return urljoin(settings.ECOMMERCE_PUBLIC_URL_ROOT, self.config.single_course_checkout_page)
+
+    def checkout_page_url(self, sku):
+        """ Construct the URL to the ecommerce checkout page and include a product. """
+        return "{}?sku={}".format(self.payment_page_url(), sku)
+
+    def register_then_add_to_cart_path(self, course_id, sku):
```

Comment:
Can we be consistent about using `url` or `path` in method names, unless there is a subtle distinction that I am missing.  I prefer `url` personally, but either is okay as long as it's consistent.